mathesar/api/db/viewsets/columns.py

```diff
@@ -92,7 +92,9 @@ def create(self, request, table_pk=None):
             except InvalidTypeError as e:
                 raise database_api_exceptions.InvalidTypeCastAPIException(
                     e,
-                    message='This type casting is invalid.',
+                    message=f'{e.column_name} cannot be casted to {e.new_type}.'
```

Comment:
```suggestion
                    message=f'{e.column_name} cannot be cast to {e.new_type}.'
```

This is the proper conjugation.